keps/0007-diagnostic-template.md

```diff
@@ -0,0 +1,43 @@
+---
+kep-number: 7
+title: Diagnostic Template
+authors:
+  - "@gerred"
+  - "@meichstedt"
+owners:
+  - "@gerred"
+  - "@meichstedt"
+creation-date: 2019-04-11
+last-updated: 2019-04-11
+status: provisional
+---
+
+# KUDO Diagnostic Template
+
+- [KUDO Diagnostic Template](#kudo-diagnostic-template)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+
+## Summary
+
+The KUDO Diagnostic Template is intended to provide an opinionated template for issues when using frameworks for KUDO and creating frameworks with KUDO. This is intended to supersede `ISSUE_TEMPLATE.md` and provide an opinionated set of diagnostic information, logs, and tooling required to adequately diagnose and respond to issues with KUDO.
+
+## Motivation
+
+Engineering teams have many incoming priorities on top of work defined upfront during sprint planning. All time spent triaging issues takes away from estimating, prioritizing, and fixing issues. This KEP provides a framework for quickly triaging and identifying issues within KUDO.
+
+### Goals
+
+* Define criteria for submitting KUDO issues
+* Define tooling, logs, and metrics needed to support, diagnose, and fix issues reported
```

Comment:
Before we can do that we would need a keep that talks about these type of diagnose bundles, e.g. how metrics/logs are produced and consumed. I would see that as a precedence to this KEP.